README.md

```diff
@@ -25,35 +23,39 @@ Other available stack variants:
 
 ## Contents
 
-1. [Requirements](#requirements)
```

Comment:
Redesigning the entire Table of Contents is out of this PR's scope.

Reply:
Sorry about that, it was not an intended change.  VS Code markdown editor auto-updated TOC.  Fixed.